src/pages/Detail/CommonDetail/ProductContentBottom.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from 'react';
+import * as P from './ProductDetailStyle';
+import { BiPlusMedical } from 'react-icons/bi';
+import { ImMinus } from 'react-icons/im';
+
+export default function ProductContentBottom({ detailContent }) {
+  const [count, setCount] = useState(1);
+
+  const handleQuantity = type => {
+    if (type === 'plus') {
+      setCount(count + 1);
+    } else if (type === 'minus') {
+      if (count === 1) {
+        return;
+      }
+      setCount(count - 1);
+    }
+  };
+
+  return (
+    <P.ProductContentBottom>
+      <P.ProductQuantity>
+        <P.QuantityContent>총 수량</P.QuantityContent>
+        <P.QuantityCountBox>
+          <ImMinus className="icon" onClick={() => handleQuantity('minus')} />
+          <P.QuantityCount>{count}</P.QuantityCount>
+          <BiPlusMedical
+            className="icon"
+            onClick={() => handleQuantity('plus')}
+          />
```

Comment:
```suggestion
          <ImMinus className="icon" onClick={() => handleQuantity(-1)} />
          <P.QuantityCount>{count}</P.QuantityCount>
          <BiPlusMedical className="icon" onClick={() => handleQuantity(1)} />
```

위의 함수 방식이라면 이렇게 구현할 수 있겠습니다!